File: app/src/main/java/com/jihoonyoon/soo/notepad/database/ScheduleDao.java
package com.jihoonyoon.soo.notepad.database;

import com.jihoonyoon.soo.notepad.models.Schedule;
import com.jihoonyoon.soo.notepad.models.Schedule_Table;
import com.raizlabs.android.dbflow.sql.language.SQLite;

import java.util.Calendar;
import java.util.List;

public class ScheduleDao {

    public static Schedule getScheduleByCalendar(Calendar calendar){
        return SQLite.select().from(Schedule.class).where(Schedule_Table.calendar.is(calendar)).querySingle();
    }

    public static void removeScheduleByCalendar(Calendar calendar){
        SQLite.delete().from(Schedule.class).where(Schedule_Table.calendar.is(calendar)).execute();
    }

    public static List<Schedule> getAllSchedule(){
        return SQLite.select().from(Schedule.class).queryList();
    }
}

File: app/src/main/java/com/jihoonyoon/soo/calendar/dao/ScheduleDao.kt
//package com.jihoonyoon.soo.calendar.dao
//
//import androidx.room.*
//import com.jihoonyoon.soo.calendar.model.Diet
//import com.jihoonyoon.soo.calendar.model.Schedule
//import java.util.*
//
//@Dao
//interface ScheduleDao {
//    @Insert
//    fun insertSchedule(schedule: Schedule)
//
//    @Delete
//    fun deleteSchedule(schedule: Schedule)
//
//    @Update
//    fun updateSchedule(schedule: Schedule)
//
//    @Query("Select * From Schedule")
//    fun selectSchedule(): List<Schedule>
//
//    @Query("Select * From Schedule Where calendar = :calendar")
//    fun selectScheduleByCalendar(calendar: Calendar): Schedule
//}